lightning/src/ln/channelmanager.rs

```diff
@@ -8285,8 +8285,11 @@ macro_rules! create_offer_builder { ($self: ident, $builder: ty) => {
 		let entropy = &*$self.entropy_source;
 		let secp_ctx = &$self.secp_ctx;
 
+		// TODO: Introduce a parameter to allow adding more paths to the created Offer.
```

Comment:
Let's drop this TODO since the exact interface for this is still TBD. Same for refunds.

Reply:
Makes sense!
Addressed in [4461115](https://github.com/lightningdevkit/rust-lightning/commit/4461115e88f7fb3e0a366a24aba8a32bc6c7b408)
Thanks!